templates/CRM/common/displaySearchCriteria.tpl

```diff
@@ -36,10 +36,10 @@
     {else}
         {foreach from=$orClauses name=criteria item=item}
             <div class="qill">
-            {$item|escape}
+            {$item|smarty:nodefaults|escape}
             {if !$smarty.foreach.criteria.last}
                 {if !empty($operator)}
-                  <span class="font-italic">...{$operator|escape}...</span>
+                  <span class="font-italic">...{$operator|smarty:nodefaults|escape:html}...</span>
```

Comment:
I don't think you need to do the :html because I think that is the default mode with the escape function iirc